docs/getting-started/GETTING_STARTED.md

```diff
@@ -392,3 +392,458 @@ or
+
+## Merge
+
+As our penultimate transformation, we call a database-agnostic merge function:
+
+```python
+# Merge the joined data into our reporting table, based on the order_id .
+# If there's a conflict in the customer_id or customer_name then use the ones from
+# the joined data
+reporting_table = aql.merge(
+    target_table=Table(
+        name=SNOWFLAKE_REPORTING,
+        conn_id=SNOWFLAKE_CONN_ID,
+    ),
+    source_table=joined_data,
+    target_conflict_columns=["order_id"],
+    columns=["customer_id", "customer_name"],
+    if_conflicts="update",
+)
+```
+
+
+## Dataframe transformation
+
+As an illustration of the `@aql.dataframe` decorator, we show a simple dataframe operation:
+
+```python
+@aql.dataframe
+def transform_dataframe(df: DataFrame):
+    purchase_dates = df.loc[:, "purchase_date"]
+    print("purchase dates:", purchase_dates)
+    return purchase_dates
+```
+
+
+After all that, you'll find the meager output of this example in the logs of the final task:
+![log](../images/task-logs.png)
+
+To view this log output, in the tree view of the DAG, click on the green box next to
+`transform_dataframe` and then on "Log" button.
+
+
+## Clean up temporary tables
+
+Finally, if there were temporary tables (`table.temp=True` or an unnamed tables) in the DAG,
+the user should consider cleaning them up by the end of the DAG run.
+
+It is possible to achieve this using one of the following approaches:
+
+(1) Clean all temporary tables by the end of the DAG run
+
+```python
+# Delete all temporary
+aql.cleanup()
+```
+
+(2) Specify a subset of temporary tables to be deleted
+
+The user may also opt for explicitly setting a subset of tables to be deleted,
+by using one of the following
+
+```python
+aql.cleanup([orders_data, joined_data])
+```
+
+or
+```python
+[orders_data, joined_data] >> aql.cleanup()
+```
+
+In all scenarios, even if the user gives a non-temporary table, only temporary
+tables will actually be deleted.
+
+## Configuring default schema
+
+We can configure the default schema that will be used for all operation involving database.
+environment variable :
+```shell
+AIRFLOW__ASTRO_SDK__SCHEMA="tmp"
+```
+or by updating Airflow's configuration
+```editorconfig
+[astro_sdk]
+schema = "tmp"
+```
+We can further configure schema per database level, this is useful for cases where there are multiple dags or same
+dag which interacts with multiple databases.
+
+### postgres
+environment variable
+```shell
+AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA="tmp"
+```
+or Airflow's configuration
+```
+[astro_sdk]
+postgres_default_schema = "tmp"
+```
+
+### snowflake
+environment variable
+```shell
+AIRFLOW__ASTRO_SDK__SNOWFLAKE_DEFAULT_SCHEMA="tmp"
+```
+or Airflow's configuration
+```
+[astro_sdk]
+snowflake_default_schema = "tmp"
+```
+
+### bigquery
+environment variable
+```shell
+AIRFLOW__ASTRO_SDK__BIGQUERY_DEFAULT_SCHEMA="tmp"
+```
+or Airflow's configuration
+```
+[astro_sdk]
+bigquery_default_schema = "tmp"
+```
```

Comment:
It's a bit strange that all these lines are being introduced as new in this PR. They should be part of the existing tutorial.